server/meca/routes.js

```diff
@@ -37,7 +37,14 @@ module.exports = app => {
 
     Manuscript.updateStatus(manuscriptId, status)
       .then(() => res.sendStatus(204))
-      .then(() => {
+      .catch(err => {
+        logger.error('Failed to process MECA callback', {
+          manuscriptId,
+          error: err.message,
+        })
+        res.status(500).send({ error: err.message })
+      })
+      .finally(() => {
         if (status === Manuscript.statuses.MECA_IMPORT_FAILED) {
           return mailer.send({
```

Comment:
Could this be placed in the `catch` after line 45?

Reply:
My miss-understanding this is a catch for updating the status of the manuscript. This makes sense.